### PR TITLE
docs: document how to enabled Google and GitHub social login

### PR DESCRIPTION
**Pull request by n1ru4l**

### CODE REVIEW

File: packages/web/docs/pages/self-hosting/oidc-login.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+By default GraphQL Hive allows you to login using a email and password.
+However, you can also enable OIDC (often refered to as social login).
+This allows you to login using a third party provider such as GitHub and Google.
+
+## GitHub
+
+First, you need to create a new GitHub application on https://github.com/settings/applications/new
+
+For the `Homepage URL` option full in the web app base url (e.g. `https://myapp.com`).
```
Comment (n1ru4l) [inline packages/web/docs/pages/self-hosting/oidc-login.mdx:11]:
```suggestion
For the `Homepage URL` option full in the web app base url (e.g. `https://graphql-hive.com`).
```

File: packages/web/docs/pages/self-hosting/oidc-login.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+For the `Authorization callback URL` option use the web app base url followed by `/auth/callback/github` (e.g. `https://myapp.com/auth/callback/github`)`.
```
Comment (n1ru4l) [inline packages/web/docs/pages/self-hosting/oidc-login.mdx:12]:
```suggestion
For the `Authorization callback URL` option use the web app base url followed by `/auth/callback/github` (e.g. `https://app.graphql-hive.com/auth/callback/github`)`.
```

File: packages/web/docs/pages/self-hosting/oidc-login.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+Now the Sign Up with GitHub button should be visible on the login page.
+
+## Google
+
+First, you need to create a new Google OIDC application following this guide: https://support.google.com/cloud/answer/6158849
+
+For the `Homepage URL` option full in the web app base url (e.g. `https://myapp.com`).
+For the `Authorization callback URL` option use the web app base url followed by `/auth/callback/google` (e.g. `https://myapp.com/auth/callback/google`)`.
```
Comment (n1ru4l) [inline packages/web/docs/pages/self-hosting/oidc-login.mdx:31]:
```suggestion
For the `Homepage URL` option full in the web app base URL (e.g. `https://app.graphql-hive.com`).
For the `Authorization callback URL` option use the web app base URL followed by `/auth/callback/google` (e.g. `https://app.graphql-hive.com/auth/callback/google`)`.
```

File: packages/web/docs/pages/self-hosting/oidc-login.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+By default GraphQL Hive allows you to login using a email and password.
+However, you can also enable OIDC (often refered to as social login).
+This allows you to login using a third party provider such as GitHub and Google.
+
+## GitHub
+
+First, you need to create a new GitHub application on https://github.com/settings/applications/new
+
+For the `Homepage URL` option full in the web app base url (e.g. `https://app.graphql-hive.com`).
+For the `Authorization callback URL` option use the web app base url followed by `/auth/callback/github` (e.g. `https://app.graphql-hive.com/auth/callback/github`)`.
```
Comment (n1ru4l) [inline packages/web/docs/pages/self-hosting/oidc-login.mdx:12]:
```suggestion
For the `Homepage URL` option full in the web app base URL (e.g. `https://app.graphql-hive.com`).
For the `Authorization callback URL` option use the web app base URL followed by `/auth/callback/github` (e.g. `https://app.graphql-hive.com/auth/callback/github`)`.
```

**Review by kamilkisiela — APPROVED** [review]
I guess the images might be too big, but let's see how it looks on staging

**Pull request merged**